src/test/java/com/alkemy/ong/util/TestEntitiesCreation.java

```diff
@@ -0,0 +1,34 @@
+package com.alkemy.ong.util;
```

Comment:
I think is a good idea to have the creation of mock entities in a util package! thanks!